ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -148,7 +159,9 @@ export default defineComponent({
       }),
       reviewConfirmText: computed((): string => {
         return localState.isReviewMode ? 'Register Changes and Pay' : 'Review and Confirm'
-      })
+      }),
+      options,
+      showCancelDialog
```

Comment:
Oh, you did define in localState. Why not the values as well?

Reply:
good point...